_im-plugin/index-alias.md

```diff
@@ -272,3 +272,58 @@ DELETE index-1/_alias/alias1
 {% include copy-curl.html %}
 
 After running the request, `alias1` no longer refers to `index-1` but still refers to `index-2`.
+
+## Overlapping field names
```

Comment:
**Technical reviewer:** Please confirm this example is relevant to an OpenSearch user and addresses the following user feedback: `Undocumented behavior for when two indexes with overlapping field names are associated with the same alias. What happens when these are different types? There appears to be some kind of implicit casting that occurs`.  I tested the example using Dev Tools. If another example is more appropriate, please replace the draft example with your example. Thank you.

Reply:
Any problem case they provided? I'm not sure what is this implicit casting they are refering to.

Reply:
> Quote reply

@bowenlan-amzn https://forum.opensearch.org/t/help-with-text-field-mapping-and-implicit-conversion-of-integer-and-boolean-to-text/16226